packages/exec/src/toolrunner.ts

```diff
@@ -524,6 +524,16 @@ export class ToolRunner extends events.EventEmitter {
           resolve(exitCode)
         }
       })
+
+      if (this.options.stdin) {
+        if (!cp.stdin) {
+          throw new Error('child process missing stdin')
+        }
+
+        cp.stdin.setDefaultEncoding('utf-8')
+        cp.stdin.write(this.options.stdin)
+        cp.stdin.end()
```

Comment:
Nit: end allows us to write write a chunk, we don't seem to need the `stdin.write` call

Reply:
I seem to recall having issues with that, but I don't remember why. Updated.